e2e/fixtures/pdfs/generate-pdfs.sh

```diff
@@ -0,0 +1,175 @@
+#!/bin/bash
+# Script to generate test PDFs with unique verification codes
+# Each PDF contains embedded text that can be used to verify end-to-end AI processing
+#
+# TODO: Dynamically randomize verification codes at generation time so fixtures can't be
+#       guessed in advance. This would involve:
+#       1. Generating a random code for each PDF at runtime
+#       2. Writing both PDF and JSON metadata with the randomized code
+#       3. Reading the code from the JSON in tests instead of hard-coding
+#       This ensures the "needle" in the PDF cannot be known before execution.
+
+set -e
+
+echo "Generating test PDFs with verification codes..."
+echo
+
+# Small PDF (33KB) - Text only with minimal content
+echo "Creating small.pdf with code SMALL-7X9Q2..."
+convert -size 800x600 xc:white \
+  -pointsize 36 -gravity center \
+  -annotate +0-150 'SMALL PDF TEST' \
+  -annotate +0-80 'Verification Code:' \
+  -fill red -pointsize 42 \
+  -annotate +0-20 'SMALL-7X9Q2' \
```

Comment:
nit: generate these arbitrary values at the top in a var and re-use them instead of hard-coding multiple times